### PR TITLE
KIALI-890 make arrow color same as edge color

### PR DESCRIPTION
**Pull request by jshaughn**

### CODE REVIEW

File: src/components/CytoscapeGraph/graphs/GraphStyles.ts [hunk 1]
```diff
@@ -124,9 +124,25 @@ export class GraphStyles {
             return ele.data('isUnused') ? 'dotted' : 'solid';
           },
           'target-arrow-shape': 'vee',
+          'target-arrow-color': (ele: any) => {
```
Comment (mtho11) [inline src/components/CytoscapeGraph/graphs/GraphStyles.ts]: You can extract this into a `const` function and use it in both places.

File: src/components/CytoscapeGraph/graphs/GraphStyles.ts [hunk 2]
```diff
@@ -124,9 +124,25 @@ export class GraphStyles {
...
+            // TODO: If there is a way to calculate this color one time and use it both for line-color
+            // amd target-arrow-color that would be nice.  The code below is a duplicate of that above.
+            const rate = ele.data('rate') ? parseFloat(ele.data('rate')) : 0;
+            if (rate === 0 || ele.data('isUnused')) {
+              return PfColors.Black;
+            }
+            const pErr = ele.data('percentErr') ? parseFloat(ele.data('percentErr')) : 0;
+            // todo: these thresholds should come from somewhere global
```
Comment (mtho11) [inline src/components/CytoscapeGraph/graphs/GraphStyles.ts]: Why don't you add these thresholds to `src/config.ts`

**Comment by jshaughn** [comment]
@mtho11 like at burger king, have it your way, changes pushed :)

File: src/components/CytoscapeGraph/graphs/GraphStyles.ts [hunk 3]
```diff
@@ -9,6 +10,22 @@ export class GraphStyles {
   }
 
   static styles() {
+    const getEdgeColor = (ele: any): string => {
+      const rate = ele.data('rate') ? parseFloat(ele.data('rate')) : 0;
+      if (rate === 0 || ele.data('isUnused')) {
+        return PfColors.Black;
+      }
+      const pErr = ele.data('percentErr') ? parseFloat(ele.data('percentErr')) : 0;
+      // todo: these thresholds should come from somewhere global
```
Comment (jmazzitelli) [inline src/components/CytoscapeGraph/graphs/GraphStyles.ts]: Isn't config() that "somewhere global"? In other words, can this comment go away?

Comment (jshaughn) [reply src/components/CytoscapeGraph/graphs/GraphStyles.ts]: flibbedy floo...

**Pull request merged**